src/components/Select/Select.tsx

```diff
@@ -120,7 +120,7 @@ const select = ({
           id={componentId}
           name={name ? name : 'select'}
           value={value}
-          className={theme.input}
+          className={theme.select}
           disabled={disabled}
```

Comment:
Are we supporting 2 kind of 'select', I see one as 'Select' & another one as 'select'.

Reply:
@ankitsaraogi yes that's kinda confusing, one style is in delicious theme & other is in component folder, so I am not changing any of them as that can cause any conflict